tests/contrib/test_cache.py

```diff
@@ -88,7 +91,7 @@ def test_generic_set_many(self, c):
 
     def test_generic_expire(self, c, fast_sleep):
         assert c.set('foo', 'bar', 1)
-        fast_sleep(2)
```

Comment:
Because these timings are spotty, sometimes it would pass, sometimes it won't because the timing was just off. I think it also has to do with how uWSGI handles garbage collection in it's cache, as it does not directly remove stuff, but waits a while and sweeps it clean every x seconds.